sources/packages/web/src/components/common/students/StudentProfile.vue

```diff
@@ -109,6 +109,52 @@
         /></v-col>
       </v-row>
     </content-group>
+    <h3 class="category-header-medium mt-4">Legacy Match</h3>
+    <content-group v-if="studentDetail.legacyProfile">
+      <v-row>
+        <v-col
+          ><title-value
+            propertyTitle="Given names"
+            :propertyValue="studentDetail.legacyProfile.firstName"
+          />
+        </v-col>
+        <v-col
+          ><title-value
+            propertyTitle="Last name"
+            :propertyValue="studentDetail.legacyProfile.lastName"
+          />
+        </v-col>
+        <v-col>
+          <title-value
+            propertyTitle="Date of birth"
+            :propertyValue="
+              dateOnlyLongString(studentDetail.legacyProfile.dateOfBirth)
+            "
+        /></v-col>
+      </v-row>
+      <v-row>
+        <v-col
+          ><title-value
+            propertyTitle="SIN"
+            :propertyValue="sinDisplayFormat(studentDetail.legacyProfile?.sin)"
+          />
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col
+          ><banner
+            v-if="studentDetail.legacyProfile?.hasMultipleProfiles"
```

Comment:
We can have the condition at `v-row` avoiding the row formation.

Reply:
Moved.